src/modules/Builder/features/ActivityFlow/ActivityFlow.utils.tsx

```diff
@@ -32,3 +34,20 @@ export const getFlowsItemActions = ({
 ];
 
 export const getActivityFlowKey = (flow: ActivityFlowFormValues) => flow.id ?? flow.key ?? '';
+
+export const getDuplicatedActivityFlow = (flow: ActivityFlowFormValues, name: string) => {
+  const duplicatedItems =
+    flow.items?.map((item) => ({
+      ...item,
+      id: undefined,
+      key: uuidv4(),
+    })) ?? [];
```

Comment:
Why  ?? not ||

Reply:
`??` is more strict for check. That why I point that only `undefined` or `null` can be there and I used `[]` as fallback? In case of `||` I suppose that result can be any of values: 0, ``, null, undefined, but I didn't use it because it cannot be '' or 0